docs/docs/apis/openidoauth/endpoints.mdx

```diff
@@ -588,4 +588,4 @@ If both parameters are provided, they must be equal.
 
 ## OAuth 2.0 Metadata
 
-**ZITADEL** does not yet provide a OAuth 2.0 Metadata endpoint but instead provides a [OpenID Connect Discovery Endpoint](#OpenID_Connect_1_0_Discovery).
+**ZITADEL** does not yet provide a OAuth 2.0 Metadata endpoint but instead provides a [OpenID Connect Discovery Endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html).
```

Comment:
I think this should have been an internal link that was broken.

```suggestion
**ZITADEL** does not yet provide a OAuth 2.0 Metadata endpoint but instead provides a [OpenID Connect Discovery Endpoint](#openid-connect-10-discovery).
```

Reply:
The thing is, you are already on the pge which that internal link was linking to. I think that’s why it effectively did nothing. But the expectation was that it would be external, since it is a «read more» about the info given on the page you’re on.

Reply:
Hm, the link in https://docs-q3cx92g22-zitadel.vercel.app/docs/apis/openidoauth/endpoints#openid-connect-10-discovery should lead to the spec. 

Does it not make more sense to jump to that heading?

Reply:
Yes, it should.

My expectation was to be taken to the spec, not the top of the page again, but maybe that's just me. Both could probably work.

Reply:
Well lets use your proposal.